modules/Microsoft.DevTestLab/labs/artifactSources/deploy.bicep

```diff
@@ -0,0 +1,87 @@
+@sys.description('Conditional. The name of the parent lab. Required if the template is used in a standalone deployment.')
+param labName string
```

Comment:
It there a reason it's using the term "parent lab" in the description? Is the "artifactSource" an old term?